RecoTracker/LSTCore/interface/alpaka/LSTESData.h

```diff
@@ -4,15 +4,14 @@
 #include "RecoTracker/LSTCore/interface/alpaka/Constants.h"
 #include "RecoTracker/LSTCore/interface/alpaka/EndcapGeometryBuffer.h"
 #include "RecoTracker/LSTCore/interface/alpaka/Module.h"
+#include "RecoTracker/LSTCore/interface/PixelMap.h"
 
 #include "HeterogeneousCore/AlpakaInterface/interface/CopyToDevice.h"
 
 #include <filesystem>
 #include <memory>
 
-namespace SDL {
-
-  struct pixelMap;
+namespace ALPAKA_ACCELERATOR_NAMESPACE::SDL {
```

Comment:
if you don't nest the namespaces here, the `ALPAKA_ACCELERATOR_NAMESPACE` will not be necessary explicitly below.
Or does the `cms::alpakatools::CopyToDevice` need to be in the global (instead of inside `ALPAKA_ACCELERATOR_NAMESPACE`)?

Reply:
I'm getting rid of this nesting. I'll push some new commits in a little bit.